pkg/kubelet/dockertools/manager.go

```diff
@@ -369,11 +370,29 @@ func (dm *DockerManager) inspectContainer(dockerID, containerName, tPath string,
 	}
 
 	glog.V(4).Infof("Container inspect result: %+v", *inspectResult)
+
+	// Get restartCount from docker label, and add into the result.
+	// If there is no restart count label in an container:
+	//	1. It is an infraContainer, it will never use restart count.
+	//	2. It is an old container or an invalid container, we just set restart count to 0 now.
+	//	TODO (random-liu) Figure out how to deal with old containers with no restart count label.
+	var restartCount int
+	if restartCountString, found := inspectResult.Config.Labels[kubernetesContainerRestartCountLabel]; found {
+		restartCount, err = strconv.Atoi(restartCountString)
+		if err != nil {
+			// (random-liu) What's the log level here?
+			glog.Errorf("Error parsing RestartCount of container %s: %v,", dockerID, restartCountString)
```

Comment:
s/RestartCount/restart count

You should print the actual error here:
`glog.Errorf("Error parsing restart count string %s for container %s: %v,", restartCountString, dockerID, err)